src/EditorFeatures/CSharpTest/AddBraces/AddBracesTests.cs

```diff
@@ -490,13 +960,18 @@ public void Dispose()
     {
         throw new NotImplementedException();
     }
-}");
+}",
+                (PreferBracesPreference)bracesPreference,
+                expectDiagnostic);
         }
 
-        [Fact, Trait(Traits.Feature, Traits.Features.CodeActionsAddBraces)]
-        public async Task FireForUsingWithoutBracesNestedInUsing()
+        [Theory, Trait(Traits.Feature, Traits.Features.CodeActionsAddBraces)]
+        [InlineData((int)PreferBracesPreference.None, false)]
+        [InlineData((int)PreferBracesPreference.WhenMultiline, false)]
+        [InlineData((int)PreferBracesPreference.Always, true)]
+        public async Task FireForUsingWithoutBracesNestedInUsing(int bracesPreference, bool expectDiagnostic)
         {
-            await TestInRegularAndScriptAsync(
+            await TestMissingOrInRegularAndScriptAsync(
             @"
 class Program
 {
```

Comment:
do we have a similar test on the outer using?

Reply:
➡️ `DoNotFireForUsingWithChildUsing`